amp.manifest

```diff
@@ -0,0 +1,25 @@
+versionAmp           = latest
+versionInfluxDB      = 1.1.3
+versionKapacitor     = 1.1.3
+versionTelegraf      = 1.1.2
+versionGrafana       = 1.1.3
+versionElasticsearch = 5.1.2
+versionEtcd          = 3.1.0
+versionNats          = v0.3.8
+versionHaproxy       = 1.0.5
```

Comment:
we should tag (annotated tag) the amp repo commit related to this version of haproxy. Right now there is no way to know which code is related to the image pushed on https://hub.docker.com/r/appcelerator/haproxy/ with tag 1.0.5.